eosio/create_accounts.sh

```diff
@@ -18,74 +14,87 @@ store_env() {
 
 account_maker() {
     # echo creating account $1
-    echo -e "${GREEN}-----------------------creating account $1-----------------------${NC}"
+    echo -e "${GREEN}----creating account $1----${NC}"
     creation_response="$(curl -s "$faucet/create/$1")"
     if [ "$(from_json .success "$creation_response")" = false ]; then
         echo "failed creation: $creation_response" 1>&2
         exit 1
     fi
     # import keys and store them just in case something went wrong
     owner_private_key="$(from_json .data.account.owner.privateKey "$creation_response")"
-    store_env owner_private_key $owner_private_key
+    store_env $1_owner_private_key $owner_private_key
     active_private_key="$(from_json .data.account.active.privateKey "$creation_response")"
-    store_env active_private_key $active_private_key
+    store_env $1_active_private_key $active_private_key
     active_public_key="$(from_json .data.account.active.publicKey "$creation_response")"
-    store_env active_public_key $active_public_key
-    kleos wallet import --private-key "$owner_private_key"
+    store_env $1_active_public_key $active_public_key
+    kleos wallet import --private-key "$owner_private_key" #qs: to where the key is imported? to default wallet of keosd on the remote keosd
     kleos wallet import --private-key "$active_private_key"
-    echo -e "${GREEN}-----------------------account created $1-----------------------${NC}"
+    echo -e "${GREEN}----account created $1----${NC}"
     # echo account created $1
 }
 
 account_funder() {
     echo funding account $1
-    echo -e "${GREEN}-----------------------funding account $1-----------------------${NC}"
+    echo -e "${GREEN}----funding account $1----${NC}"
     funding_response="$(curl -s "$faucet/get_token/$1")"
     if [ "$(from_json .success "$funding_response")" = false ]; then
         echo "failed funding: $funding_response" 1>&2
         exit 1
     fi
     kleos system buyram $1 $1 "10.0000 EOS" -p $1@active
-    echo -e "${GREEN}-----------------------account funded $1-----------------------${NC}"
-    # echo account funded $1
+    echo -e "${GREEN}----account funded $1----${NC}"
+    echo account funded $1
 }
 
 # create bbs account if not exist and fund it.
 if [ ! "$bbs_account" ]; then
-    bbs_account="$(name_maker)"
-    store_env bbs_account $bbs_account
+    read -p "No bbs_account was configured. Do you want to create a new one (yes/no)?" ans
+    if [[ "yes" == $ans ]]; then
```

Comment:
If `ans` is empty this will error (try it). Basically, what the shell sees in such a case is:
```sh
    if [[ "yes" == ]]; then
```
To avoid this problem, just put the var in double quotes.

Reply:
I don't get any error when just press "enter" for example and ans is empty.

anyway this is probably a good practice.